connect_ext_ppr/db.py

```diff
@@ -220,7 +220,7 @@ def get_cbc_extension_db(engine: Engine = Depends(get_cbc_extension_db_engine)):
     )
     db: Session = SessionMaker(bind=engine)
     try:
-        yield db
+        return db
```

Comment:
[COD-C] why do we need this change?

Reply:
Before it was used as a FastAPI `Depends` https://github.com/cloudblue/connect-extension-xvs/blob/36f9c585de488e7d36c8f5e1cc1662b07cd83346/connect_ext_ppr/webapp.py#L812

Now it is used only as a normal function and it doesn't work currently
https://github.com/cloudblue/connect-extension-xvs/blob/36f9c585de488e7d36c8f5e1cc1662b07cd83346/connect_ext_ppr/tasks_manager.py#L36